### ISSUE TITLE
loadhint is not defined

### ISSUE THREAD

**Issue by ceylanb** [CTFd/CTFd#1225]
**Environment**:

  - CTFd Version/Commit: 2.2.3
  - Operating System: Archlinux
  - Web Browser and Version: Version 1.1.23 Chromium: 79.0.3945.88 (Official Build) unknown (64-bit) 

**What happened?**
Hint can be created by admin but not be seen from users. When I click the "View Hint" button an error is occurred in Javascript: 

```
Uncaught ReferenceError: loadhint is not defined
    at HTMLAnchorElement.onclick (challenges:1)
```

**What did you expect to happen?**
I expect that the users can see the hints.

**How to reproduce your issue**
I have deployed the project by "python serve.py" and run the project over the browser. 

**Any associated stack traces or error logs**
```
Uncaught ReferenceError: loadhint is not defined
    at HTMLAnchorElement.onclick (challenges:1)
```

**Comment by ColdHeat** [comment]
This is currently resolved in master and waiting for a release. Just mentioning it to let you know. I'll close this once it's out.

**Comment by ColdHeat** [comment]
This will be released in v2.3. Closing this to clear it off the milestone.
